x-pack/plugins/siem/public/components/open_timeline/note_previews/note_preview.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiAvatar, EuiFlexGroup, EuiFlexItem, EuiText, EuiSpacer, EuiToolTip } from '@elastic/eui';
+import { FormattedRelative } from '@kbn/i18n/react';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+import { FormattedDate } from '../../formatted_date';
+import { getEmptyValue } from '../../empty_value';
+import { Markdown } from '../../markdown';
+
+import * as i18n from '../translations';
+
+interface Props {
+  note?: string | null;
+  updated?: number | null;
+  updatedBy?: string | null;
+}
```

Comment:
These same three properties are used in `x-pack/plugins/siem/public/components/open_timeline/types.ts`

Would it be possible to reuse this as an interface between these two? Does that make sense? Optional suggesiton.